### DISCUSSION TITLE
Device Bound Session Credentials!!!!

### DISCUSSION THREAD

**Discussion by ghost** [moonD4rk/HackBrowserData#320] [General]
https://www.malwarebytes.com/blog/news/2024/04/google-chrome-gets-device-bound-session-credentials-to-stop-cookie-theft 

chrome and all chromium browser will most likely get this soon

as i understood it, it means cookies are bound by device with some kind of key, and that key is saved in TMP

the question is, when it is out, would this project be able to still get cookies? as i do not think the key can be fetched from TPM that easy

**Comment by moonD4rk** [comment]
Thank you very much for your reminder. I am also continuously paying attention to DBSC. If it is really as mentioned in the document, stealing Cookies will become very difficult. However, we still need to wait for Google to release the official version for more details.

**Reply by ghost** [reply]
yes,  i would be intrested to see if you simply remove the cookie module and what Google will be doing, or that we some how can hack into it
